api/types/client.go

```diff
@@ -24,6 +25,7 @@ type VanServiceInterfaceCreateOptions struct {
 	Port       int
 	TargetPort int
 	Headless   bool
+	Aggregate  string
```

Comment:
This feels logically distinct from the console server. If including it I would also include the EventChannel attribute?

Reply:
Agreed that it doesn't need to be in this patch set. Relates to the question of VanServiceInterfaceCreateOptions and ServiceInterface types. I will remove for the purposes of this PR.